audio/src/lib.rs

```diff
@@ -57,7 +57,9 @@
 //! ```
 //!
 //! We also support [wrapping][wrap] external buffers so that they can
-//! interoperate like other audio buffers.
+//! interoperate like other audio buffers. Library authors using the [Buf]/[BufMut]
+//! traits as generic inputs to functions should re-export the [wrap] module so
+//! users of the library are not required to use this crate's buffer structs.
```

Comment:
Library authors should generally not need to re-export the `wrap` module. They probably shouldn't depend on anything but `audio_core`. The `wrap` module could be used by users of an audio crate to wrap buffers.